lab1/README.md

```diff
@@ -1 +1,38 @@
-#Lab 1
+#Lab 1: WikiMVC
+
+##Intro
+This lab will have 3 components: 
+- Choosing a client-side Javascript framework and learning it 
+- Constructing a wiki clone with certain functionality in that framework
+- Preparing a presentation to the class about what your chosen framework
+
+The lab will be completed in pairs. You must collaborate on github repo separate from either of your olinjs repos.
+
+##Choosing a Framework
+Each pair will be choosing a client side framework. Angular, Backbone (potentially with Marionette), Ember, React, Flight, Knockout, an Riot are all valid options but there are many other options.
+If you are unsure of a framework qualifies just check with us and we should be able to tell you very quickly. 
+
+##Constructing a Wiki
+You must construct a wikipedia-like clone using your chosen framework. You should program the app using the conventions, styles, and functionality of your chosen framework, show off what it can do. It should have the following functionality:
+- List out all pages on various topics that have pages
+- Read any submitted pages and their content
+- Allow users to submit new pages on topics
+- Allow users to edit existing pages 
+- Documentation for each major component (Minimum of few sentences at top of every file)
+- Not look terrible
+- One other feature you'd like to implement (Could be as simple as a random page or something more serious like searching, but we are urging creativity)
+
+
+If you want to make the wiki have a special topic area (like harry potter or some game) you are encouraged to do so. 
+Brownie points will be awarded for awesome looking websites whose themes match the topic area, though having all the functionallity should come before making it pretty.
+
+##Framework Presentation
+You and your partner will get together with the other groups who are doing the same framework and create a 8-10 minute presentation about the framework. 
+The goal of the presentation is NOT to teach the class to use the framework. 
+Rather the presentation should contain the following pieces:
+- Basic overview and introduction to the framework
+- Any unique features that you believe are worth showing off
+- What are some advantages of the framework
+- What are some disadvantages / difficulties of the framework
+- What type of project would the framework be suited or not suited for
```

Comment:
"Is there a specific type of project that the framework would work or not work well for?"